autogen/main/cluster.tf.tmpl

```diff
@@ -307,6 +307,13 @@ resource "google_container_cluster" "primary" {
     }
   }
 {% endif %}
+
+  notification_config {
```

Comment:
Looks like `notification_config` is in beta, could you move this to into beta conditional block and regenerate? 
```
{% if beta_cluster %}
notification_config {
...
}
{% endif %}
```

Reply:
Done.